.github/workflows/pr-builds.yaml

```diff
@@ -78,26 +65,3 @@ jobs:
       SignExecutable: false
       BuildInnoSetup: true
       BuildArchive: true
-
-  comment-build-end:
-    if: github.event.pull_request.draft == false
-    needs: [ build-linux, build-windows ]
-    permissions:
-      pull-requests: write
-
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Comment PR
-        uses: thollander/actions-comment-pull-request@v2
-        with:
-          message: |
-            ### Builds
-            Commit: `${{ github.sha }}`
-
-            * [Windows InnoSetup](${{ needs.build-windows.outputs.ArtifactUrlWindowsInnoSetup }})
-            * [Windows Archive](${{ needs.build-windows.outputs.ArtifactUrlWindowsArchive }})
-            * [Linux AppImage](${{ needs.build-linux.outputs.ArtifactUrlLinuxAppImage }})
-          comment_tag: builds
```

Comment:
Something like this wouldn't work due to permissions then?
```yaml
 comment:
    if: github.event.pull_request.draft == false
    needs: [ build-linux, build-windows ]
    runs-on: ubuntu-latest

    steps:
      - name: Comment PR
        uses: thollander/actions-comment-pull-request@master
        with:
          message: |  
            ### Builds
            Commit: `${{ github.sha }}`

            * [Windows InnoSetup](${{ needs.build-windows.outputs.ArtifactUrlWindowsInnoSetup }})
            * [Windows Archive](${{ needs.build-windows.outputs.ArtifactUrlWindowsArchive }})
            * [Linux AppImage](${{ needs.build-linux.outputs.ArtifactUrlLinuxAppImage }})
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Yes, doesn't work. The `GITHUB_TOKEN` doesn't have the permissions for it.

Reply:
Sadge